CONTRIBUTING.md

```diff
@@ -60,6 +60,8 @@ If you're publishing a new package, ensure that its `package.json` file contains
 	}
 ```
 
+Update the packages `CHANGELOG.md` file with the forthcoming version number (`x.y.z`) along with the notable changes included in the release.
```

Comment:
Can we add a note that it should be put under `Unpublished` section, which should get updated after release gets out? There might be one issue with this flow, updating `CHANGELOG.md` after release will make Lerna think that there needs to be another version bump :P We probably should teach Lerna to ignore this file from its checks.